src/GovUk.Education.ExploreEducationStatistics.Data.Api/Startup.cs

```diff
@@ -187,6 +190,13 @@ public void ConfigureServices(IServiceCollection services)
                     policy.Requirements.Add(new ViewSubjectDataRequirement()));
             });
 
+            var publicAppUrl = Configuration.GetValue<string>("PublicAppUrl");
```

Comment:
Not a biggy, but would consider inlining this directly into the service factory:

```cs
httpClient.BaseAddress = new Uri(Configuration.GetValue<string>("PublicAppUrl"));
```